ci/env.sh

```diff
@@ -0,0 +1,26 @@
+#smoke test environment specifications:
```

Comment:
We might want to add a way to specify kubeconfig path, it's useful when the kubeconfig is not under the default location - $HOME/.kube/config. Something like KUBECONFIG_PATH, this way we can run export KUBECONFIG=$KUBECONFIG_PATH command in the script for the oc client to pick it up and talk to the cluster. Thoughts?

The workloads need to be modified to support it as well but it might be out of the scope for this PR. See https://github.com/cloud-bulldozer/e2e-benchmarking/issues/118.